source/templates/handlebars-basics.md

```diff
@@ -52,6 +52,44 @@ controllers and components.
 
 ### Helpers
 
-Helpers bring a minimum of logic into Ember templating. Ember ships with several
-built-in helpers, which are explained in the following guides, and also allows
-you to [write your own helpers](../writing-helpers/).
+Ember gives you the ability to [write your own helpers](../writing-helpers/), to bring a minimum of logic into Ember templating.
+
+For example, let's say you would like the ability to add a few numbers together, without needing to define a computed property everywhere you would like to do so.
+
+```javascript
+// helpers/sum.js
+export default Ember.Helper.helper(function(params) {
+  return params.reduce(function(a, b) {
+    return a + b;
+  });
+});
+
+```
+
+This helper allows you to call `{{sum}}` in your templates:
+
+```html
+<p>Total: {{sum 1 2 3}}</p>
+```
+
+This will output a total of `6`.
+
+Ember ships with several built-in helpers, which you will learn more about in the following guides.
+
+#### Nested Helpers
+
+Helpers have the ability to be nested within other helper invocations and also component invocations.
+
+This gives you the flexibility to compute a value _before_ it is passed in as an argument or an attribute of another.
+
+It is not possible to nest curly braces `{{}}` inside of a helper already using the curly braces, so a "sub-expression" can be created using parentheses `()`:
```

Comment:
don't use `"sub-expression"`, doesn't explain anything and might confuse beginners.
consider a wording like "since it is not possible to nest curly braces `{{}}`, the correct way to use nested helpers is using parenthesis `()`:"